packages/stack/src/views/Stack/Card.tsx

```diff
@@ -159,12 +159,15 @@ export default class Card extends React.Component<Props> {
       spec.animation === 'spring' ? Animated.spring : Animated.timing;
 
     onTransitionStart?.({ closing: Boolean(closing) });
+    this.handleStartInteraction();
```

Comment:
small thing: can we set it before `onTransitionStart`? so user can still rely on it in `transitionStart` event